Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:8.9
+
+WORKDIR /dashboard
+
+EXPOSE 8080
```

Comment:
We don't need to expose this if we're including the `-p` flag when running.

Reply:
removed this!